### ISSUE TITLE
HIgh training accuracy and low validation accuracy

### ISSUE THREAD

**Issue by karliell** [xuczhang/tapnet#4]
Hi! I trained the model on my own dataset. I found that the training accuracy is very high. Several epoch will reach 99% accuracy. However, the validation accuracy is always 36%. 
More epoch results the loss_validation going higher. 
![image](https://user-images.githubusercontent.com/36145675/82733621-06929380-9ce3-11ea-9277-65463e47feb1.png)

I have shuffled my dataset but I don't know how to add `drop` or some other techniques to reduce overfitting. Would you please help me to give me some suggestions?

Thanks a lot!

**Comment by flash121123** [comment]
I think the best way to reduce overfitting is doing cross-validation (if you have enough data). You can adjust parameters such as kernel size, weight decay. Also you can adjust the learning rate to see whether it will affect performance.

**Comment by omrylcn** [comment]
I thin,k your train and validation dataset column indexes  can be different, it is just a prediction.
